analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -236,7 +236,18 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
                             }
                         }
 
-                        else -> VcsInfo(type = VcsType(type), url = url, revision = tag)
+                        else -> {
+                            VcsHost.fromUrl(url)?.let { host ->
```

Comment:
FYI @MarcelBochtler and @mnonnenmacher, I didn't see a need to limit the logic to GitHub in the end has `VcsHost` allows to implement this nicely in a generic way, this could very well happen also e.g. on Bitbucket, and most of the code is on GitHub anyway (so not handling the rest would not really shield us against the logic falsely kicking in).